storage/innobase/trx/trx0roll.cc

```diff
@@ -617,6 +618,7 @@ trx_savept_take(
 	trx_savept_t	savept;
 
 	savept.least_undo_no = trx->undo_no;
+	savept.vers_update_trt = trx->vers_update_trt;
```

Comment:
Why do we need both fields? And what exactly is the purpose of this field?

Reply:
Answered [above](https://github.com/MariaDB/server/pull/478#discussion_r152256282).